commands/bridge_configure.go

```diff
@@ -36,15 +35,6 @@ func runBridgeConfigure(cmd *cobra.Command, args []string) error {
 	defer backend.Close()
 	interrupt.RegisterCleaner(backend.Close)
 
-	termState, err := terminal.GetState(int(syscall.Stdin))
```

Comment:
those are still needed if it's not a pipe

Reply:
I moved this part to the github bridge configuration process because it the only place where `terminal.ReadPassword` is used